Lib/test/test_posix.py

```diff
@@ -22,6 +19,11 @@
 import textwrap
 from contextlib import contextmanager
 
+try:
+    import posix
+except ImportError:
+    import nt as posix
```

Comment:
It's a little bit surprising, but well, it's convenient for these tests :-)

The C extension should be called `_os` instead of `posix` or `nt` :-(